src/vehicle.h

```diff
@@ -1113,6 +1113,9 @@ class vehicle
             bool fullsize = false, bool verbose = false, bool desc = false,
             bool isHorizontal = false );
 
+        // Vehicle speed gauge
+        void print_speed_gauge( const catacurses::window &win, const point &, unsigned int spaceing = 0 );
```

Comment:
Our style guide suggests using int over unsigned int.

Reply:
Since it mentions also `unsigned int` and `unsigned char`, I've weighted if negative space causing things on top of each other using it.

Reply:
It'll throw invalid argument then.